website/docs/getting-started/setup/instance-management/README.mdx

```diff
@@ -0,0 +1,127 @@
+---
+
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Management
+
+## Updating to the latest release
+
+You can perform a manual update of your Appsmith instance by running the following commands in the Appsmith installation directory.
+
+```bash
+// To restart appsmith without docker compose
+docker pull appsmith/appsmith-ce && docker restart appsmith
+
+// To restart appsmith with docker compose
+docker-compose pull && docker-compose up -d --force-recreate appsmith
+```
+
+## Checkpoint upgrade
+Upgrading a Docker container checkpoint to the latest version is a crucial step in maintaining the stability, security, and performance of the containerized applications. 
+
+You may need to manually map your instance to the required version If your current version of Appsmith is less than `v1.9.2`. To do this, access your Appsmith instance and make the following change to the `docker-compose.yml` file:
```

Comment:
Replace with 

**Appsmith checkpoint versions**
         - v1.9.2

Access your Appsmith instance and make the following change to the `docker-compose.yml` file: